bitmark-references/bitmark-cli/bitmark-cli.md

```diff
@@ -0,0 +1,729 @@
+# bitmark-cli Command Reference
+
+## Basic Command Structure
+
+~~~
+bitmark-cli [global-options] command [command-options]
+~~~
+
+## Global Options
+
+Options are listed as long form with any single character abbreviation.
+
+`--verbose|-v`
+
+: This allows the CLI to output additional information for each
```

Comment:
@hxw: Does the `:` has any special meaning? They make me a little bit confused.

Reply:
it means inline code and will format with a fixed width font to represent command-line values.
I am not sure how to format this properly, so if there is a better way it should be changed